core/store/models/eth.go

```diff
@@ -292,3 +296,248 @@ func (h *Head) NextInt() *big.Int {
 	}
 	return new(big.Int).Add(h.ToInt(), big.NewInt(1))
 }
+
+// WeiPerEth is amount of Wei currency units in one Eth.
+var WeiPerEth = new(big.Int).Exp(big.NewInt(10), big.NewInt(18), nil)
```

Comment:
It would make sense to me to move this closer to `currencies`, I was looking for something like this the other day and couldn't find it

Reply:
`assets` maybe?  We have a package for that already, and this would make sense in there.

Reply:
Would like to see this move into `assets` but not gonna block on such a tiny point.